templates/common/infra/terraform/core/host/appserviceplan/appserviceplan_output.tf

```diff
@@ -0,0 +1,4 @@
+output "APPSERVICE_PLAN_ID" {

```

Comment:
I don't think we need _ between service and plan.

Reply:
I would like to keep the same style used for the rest of the modules outputs as well.  
[log analytics](https://github.com/kaizentm/azure-dev/blob/da30788c36840a5254cf70f7b78f80c5519e92c3/templates/common/infra/terraform/core/monitor/loganalytics/loganalytics_output.tf#L1), [app service plan](https://github.com/kaizentm/azure-dev/blob/da30788c36840a5254cf70f7b78f80c5519e92c3/templates/common/infra/terraform/core/host/appserviceplan/appserviceplan_output.tf#L1)